contracts/fio.perms/fio.perms.abi

```diff
@@ -0,0 +1,197 @@
+          "name": "object_type_hash",
+          "type": "uint128"
+        },
+        {
+          "name": "object_name",
+          "type": "string"
+        },
+        {
+          "name": "object_name_hash",
+          "type": "uint128"
+        },
+        {
+          "name": "permission_name",
+          "type": "string"
+        },
+        {
+          "name": "permission_name_hash",
+          "type": "uint128"
+        },
+        {
+          "name": "permission_control_hash",
+          "type": "uint128"
+        },
+        {
+          "name": "owner_account",
+          "type": "uint64"
+        },
+        {
+          "name": "auxilliary_info",
+          "type": "string"
+        }
+      ]
+    },
+    {
+       "name": "access_info",
+       "base": "",
+       "fields": [
+         {
+           "name": "id",
+           "type": "uint64"
+         },
+         {
+           "name": "permission_id",
+           "type": "uint64"
+         },
+         {
+           "name": "grantee_account",
+           "type": "uint64"
+         },
+         {
+           "name": "access_hash",
+           "type": "uint128"
+         }
+       ]
+     },
+    {
+      "name": "addperm",
+      "base": "",
+      "fields": [
+        {
+          "name": "grantee_account",
+          "type": "name"
+        },
+        {
+          "name": "permission_name",
+          "type": "string"
+        },
+        {
+          "name": "permission_info",
+          "type": "string"
+        },
+        {
+          "name": "object_name",
+          "type": "string"
+        },
+        {
+          "name": "max_fee",
+          "type": "int64"
+        },
+        {
+          "name": "tpid",
+          "type": "string"
+        },
+        {
+          "name": "actor",
+          "type": "name"
+        }
+      ]
+    },
+    {
+      "name": "remperm",
+      "base": "",
+      "fields": [
+        {
+          "name": "grantee_account",
+          "type": "name"
+        },
+        {
+          "name": "permission_name",
+          "type": "string"
+        },
+        {
+          "name": "object_name",
+          "type": "string"
+        },
+        {
+          "name": "max_fee",
+          "type": "int64"
+        },
+        {
+          "name": "tpid",
+          "type": "string"
+        },
+        {
+          "name": "actor",
+          "type": "name"
+        }
+      ]
+    },
+     {
+       "name": "clearperm",
+       "base": "",
+       "fields": [
+         {
+           "name": "permission_name",
+           "type": "string"
+         },
+         {
+           "name": "object_name",
+           "type": "string"
+         }
+       ]
+     }
+  ],
+  "actions": [
+    {
+      "name": "addperm",
+      "type": "addperm",
+      "ricardian_contract": ""
+    },
+    {
+      "name": "remperm",
+      "type": "remperm",
+      "ricardian_contract": ""
+    },
+     {
+       "name": "clearperm",
+       "type": "remperm",
```

Comment:
Use the clearperm struct?

Reply:
I have discovered this in testing as well, commit to change coming up. thanks